lib/functions.sh

```diff
@@ -104,8 +108,18 @@ function update-link {
   fi
 }
 
-function is-released-version {
-  [[ ${1} =~ ^([0-9]+\.){0,2}[0-9]$ ]] && return 0
+function clients-need-built {
```

Comment:
Fairly minor, but the grammar nerd in me feels the urge to point out that this should be "clients-need-to-be-built" to be proper ;)